haystack/pipelines/base.py

```diff
@@ -18,14 +21,51 @@
     ray = None  # type: ignore
     serve = None  # type: ignore
 
-from haystack.schema import MultiLabel, Document
+from haystack.schema import Label, MultiLabel, Document
 from haystack.nodes.base import BaseComponent
 from haystack.document_stores.base import BaseDocumentStore
 
 
 logger = logging.getLogger(__name__)
 
 
+class EvaluationResult:
```

Comment:
I am wondering whether to move this class to `haystack/schema.py` because it seems like a data primitive to me just like `Document` or `Answer` and we might reuse it in other modules.

Reply:
Yes, makes sense. Moved the `calculate_metrics()` method too.